pkg/app/piped/toolregistry/install.go

```diff
@@ -28,7 +28,7 @@ import (
 const (
 	defaultKubectlVersion   = "1.18.2"
 	defaultKustomizeVersion = "3.8.1"
-	defaultHelmVersion      = "3.2.1"
+	defaultHelmVersion      = "3.8.2"
```

Comment:
Requires Helm >= v3.8.0 because using the OCI Registry is experimental feature in earlier versions and it may cause malfunctions.
Helm follows Semantic Versioning, so backward compatibility is maintained.

Reply:
Could you move this to another PR?
It is a notable change and should be included in the release changelog.